admin/resources/js/components/ClassificationTable.tsx

```diff
@@ -1,45 +1,57 @@
 import React, { useMemo } from "react";
-import { Column } from "react-table";
 import {
   GetClassificationsQuery,
   useGetClassificationsQuery,
 } from "../api/generated";
 import { notEmpty } from "../helpers/util";
-import Table from "./Table";
+import Table, { FilterableColumn } from "./Table";
 
 export const ClassificationTable: React.FC<GetClassificationsQuery> = ({
   classifications,
 }) => {
-  const columns: Array<Column> = useMemo(
+  const columns: Array<FilterableColumn<any>> = useMemo(
     () => [
       {
         Header: "ID",
         accessor: "id",
+        showCol: true,
       },
       {
         Header: "Name",
         accessor: "name.en",
+        showCol: true,
       },
       {
         Header: "Group",
         accessor: "group",
+        showCol: true,
       },
       {
         Header: "Level",
         accessor: "level",
+        showCol: true,
       },
       {
         Header: "Minimum Salary",
         accessor: "minSalary",
+        showCol: false,
       },
       {
         Header: "Maximum Salary",
         accessor: "maxSalary",
+        showCol: false,
       },
     ],
     [],
   );
 
+  const hiddenCols: string[] = [];
+  columns.forEach((column) => {
+    if (column.showCol === false && column.accessor) {
+      hiddenCols.push(column.accessor.toString());
+    }
+  });
```

Comment:
Instead of defining FilterableColumns, only to map showCol to initialState.hiddenColumns _anyway_, lets just pass in initialState with hiddenColumns directly.